### PR TITLE
Standardize ci

### LINKED ISSUE #116: Drone setup
Move from Travis to Drone

### PR DESCRIPTION
**Pull request by skshetry**
Standardize CI for `owncloud/impersonate`

Fixes #116

### CODE REVIEW

**Comment by skshetry** [comment]
This currently requires codecov token. So, the CI is failing

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,14 @@ js/templates/*.handlebars.js
 # test files
 clover.xml
```
Comment (phil-davis) [inline .gitignore]: these lines can be removed (comment heading, ``clover.xml`` and 1 blank line)

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,312 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/impersonate
+
```
[the remark this answers is not in the document]
Comment (skshetry) [reply .drone.yml]: @phil-davis I didn't get it. There's always a new line, even in password policy.

[the remark this answers is not in the document]
Comment (phil-davis) [reply .drone.yml]: I think there were 2 blank lines, but now it is fine.

**Pull request merged**